common/test/acceptance/fixtures/catalog.py

```diff
@@ -13,31 +13,31 @@ class CatalogFixture(object):
     """
     Interface to set up mock responses from the Catalog stub server.
     """
-    def install_programs(self, programs):
+    def install_programs(self, data):
         """Set response data for the catalog's course run API."""
         key = 'catalog.programs'
 
-        requests.put(
-            '{}/set_config'.format(CATALOG_STUB_URL),
-            data={key: json.dumps(programs)},
-        )
+        if isinstance(data, dict):
+            key += '.' + data['uuid']
```

Comment:
@davec-edx I went ahead and merged to keep this from getting caught up over string concatenation. I tend to use format strings for readability unless the associated overhead doesn't add much value, in which case I just use the `+` operator.